templates/fr/complaint.txt

```diff
@@ -0,0 +1,13 @@
+À Qui Cela Peut Concerner:
+
+Je suis, par les présentes, le dépôt d'une plainte conformément à l'Article 77 GDPR. Sur {request_date}, j'ai envoyé une demande conformément à l'Article {request_article} GDPR pour le contrôleur suivant:
+{request_recipient_address}
+
+{Description des faits par le contrôleur et les étapes que vous avez pris (par exemple, injonctions ou toute autre correspondance, inclure les reçus)}. 
```

Comment:
In this one line, we are addressing the user instead of the supervisory authority. And in datenanfragen/website#148, we decided to use 'tu' for our users instead of 'vous.